src/mlpack/methods/cf/decomposition_policies/quic_svd_method.hpp

```diff
@@ -0,0 +1,174 @@
+/**
+ * @file methods/cf/decomposition_policies/quic_svd_method.hpp
+ * @author Adarsh Santoria
+ *
+ * Implementation of the quic svd method for use in
+ * Collaborative Fitlering.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_METHODS_CF_DECOMPOSITION_POLICIES_QUIC_SVD_METHOD_HPP
+#define MLPACK_METHODS_CF_DECOMPOSITION_POLICIES_QUIC_SVD_METHOD_HPP
+
+#include <mlpack/prereqs.hpp>
+#include <mlpack/methods/quic_svd/quic_svd.hpp>
+
+namespace mlpack {
+
+/**
+ * Implementation of the Quic SVD policy to act as a wrapper when
```

Comment:
```suggestion
 * Implementation of the QUIC-SVD policy to act as a wrapper when
```
Technically "QUIC" is an acronym, so it should all be capitalized, but I can't remember exactly what it stands for off the top of my head. :)  (Do you think you can change the documentation accordingly throughout?)